server/webpack.config.js

```diff
@@ -40,17 +43,19 @@ module.exports = {
   entry: {
     applicant: './app/assets/javascripts/applicant_entry_point.ts',
     admin: './app/assets/javascripts/admin_entry_point.ts',
-    uswds: './node_modules/@uswds/uswds/dist/js/uswds.min.js',
-    uswdsStyles: './app/assets/stylesheets/uswds/styles.scss',
+    uswds: [
```

Comment:
praise:  Thanks for making this cleaner!  I had tried to do this initially and, for some reason that I don't remember, it wasn't working for me.